### ISSUE TITLE
Croniter `get_prev` fails on leap year days (like today)

### ISSUE THREAD

**Issue by pvieito** [pallets-eco/croniter#62]
Croniter `get_prev` fails on leap year days (like today):

```
import croniter
import datetime

croniter.croniter("15 22 29 2 *", datetime.datetime(2024, 2, 29)).get_prev(datetime.datetime)
# raises: croniter.croniter.CroniterBadDateError: failed to find prev date
```

**Comment by kiorky** [comment]
This was already fixed in release 2.0.2 (6824882167f67c7f2e1c34d0e8c9e4fd73ba499f)

**Comment by pvieito** [comment]
@kiorky Thanks!
